share/rocm/cmake/ROCMDocs.cmake

```diff
@@ -1,7 +1,9 @@
 # ######################################################################################################################
-# Copyright (C) 2021 Advanced Micro Devices, Inc.
```

Comment:
2021-2023

Reply:
Done. (been done for a while, just didn't mark here)